src/insert.c

```diff
@@ -140,6 +141,10 @@ insert_main_table_trigger(PG_FUNCTION_ARGS)
 	}
 	PG_END_TRY();
 
+	/* add 1 to the number of processed tuples in the commandTag. Without this tuples that return 
+	 * NULL in before triggers are not counted. */
+	add_number_tuples_processed(1);
```

Comment:
I'd prefer that this "public" function was prefixed with the module where it is implemented, e.g., `executor_set_number_of_processed_tuples()`. Then one knows where to look for this  function and it will ensure no name collisions across modules. Namespace/prefixing is also in our style guide.